concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1507,53 +1507,90 @@ jobs:
+                  tag: 161376949_TF_ruby
+              inputs:
+                - name: datadog-tfstate
+                - name: paas-cf
+                - name: config
+              outputs:
+                - name: updated-tfstate
+              params:
+                TF_VAR_env: {{deploy_env}}
+                TF_VAR_datadog_api_key: {{datadog_api_key}}
+                TF_VAR_datadog_app_key: {{datadog_app_key}}
+                TF_VAR_aws_account: {{aws_account}}
+                ENABLE_DATADOG: {{enable_datadog}}
+              run:
+                path: sh
+                args:
+                  - -e
+                  - -c
+                  - |
+                    if [ -n "${TF_VAR_datadog_api_key}" ] && [ -n "${TF_VAR_datadog_app_key}" ] && [ "${ENABLE_DATADOG}" = "false" ]; then
+                      echo "Datadog disabled but keys present, running check cleanup..."
+                      terraform destroy -force \
+                         -state=datadog-tfstate/datadog.tfstate \
+                         -state-out=updated-tfstate/datadog.tfstate \
+                         paas-cf/terraform/datadog
+                      exit 0
+                    fi
+  
+                    if [ "${ENABLE_DATADOG}" = "true" ]; then
+                      terraform apply \
+                        -var-file=paas-cf/terraform/{{aws_account}}.tfvars \
+                        -state=datadog-tfstate/datadog.tfstate -state-out=updated-tfstate/datadog.tfstate \
+                        -var-file=config/job_instances.tfvars paas-cf/terraform/datadog
+                    else
+                      echo "Datadog disabled, skipping terraform run..."
+                      cp datadog-tfstate/datadog.tfstate updated-tfstate/datadog.tfstate
+                    fi
+            ensure:
+              put: datadog-tfstate
+              params:
+                file: updated-tfstate/datadog.tfstate
+  
+          - task: datadog-apply-attributes
+            config:
+              platform: linux
+              image_resource:
+                type: docker-image
+                source:
+                  repository: ruby
+                  tag: 2.2-alpine
+              inputs:
+                - name: datadog-tfstate
+                - name: paas-cf
+              params:
+                TF_VAR_env: {{deploy_env}}
+                TF_VAR_datadog_api_key: {{datadog_api_key}}
+                TF_VAR_datadog_app_key: {{datadog_app_key}}
+                TF_VAR_aws_account: {{aws_account}}
+                ENABLE_DATADOG: {{enable_datadog}}
+              run:
+                path: sh
+                args:
+                  - -e
+                  - -c
+                  - |
+                    # FIXME: Remove this task when https://github.com/zorkian/go-datadog-api/issues/56 is fixed
+                    if [ "${ENABLE_DATADOG}" = "true" ]; then
+                      gem install dogapi -v 1.24.0
```

Comment:
Can you use Bundler here so that we get the same version of the gem that's used by the tests?

As mentioned in https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/84#issuecomment-276138891, we'll need to use groups to prevent all of the test dependencies being installed.